dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04200CreateDefaultVanityURL.java

```diff
@@ -0,0 +1,235 @@
+package com.dotmarketing.startup.runonce;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+
+/**
+ * This upgrade task will insert into the tables Inode and Structure the default Vanity URL Content Type.
+ * This is needed to be able to migrate the old Vanity URL's (on the virtual_link table) to the new Vanity URL that are content.
+ * 
+ * @author erickgonzalez
+ * @version 4.2.0
+ * @since Jun 6, 2017
+ *
+ */
+public class Task04200CreateDefaultVanityURL extends AbstractJDBCStartupTask {
+
+	// TODO: If the Content Type is changed we need to check the values here are the same that the ones on the Starter
+	private final String DEFAULT_VANITY_URL_STRUCTURE_INODE = "8e850645-bb92-4fda-a765-e67063a59be0";
+	private final String DEFAULT_VANITY_URL_STRUCTURE_VARNAME = "Vanityurlasset";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "DEFAULT_VANITY_URL_STRUCTURE_VARNAME" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)